src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -161,5 +162,15 @@ public override int GetHashCode()
         {
             return !Equals(left, right);
         }
+
+        public static SuffixedList<PartValue> PartsToList(IEnumerable<global::Part> parts, SharedObjects shared)
```

Comment:
Again, same question.  What would have been supposedly inferior about doing this:

public static **ListValue**< PartValue > PartsToList(IEnumerable< global::Part > parts, SharedObjects shared)